### ISSUE TITLE
NPM  warning - deprecated querystring@0.2.0

### ISSUE THREAD

**Issue by danielpeintner** [eclipse/thingweb.node-wot#717]
`npm ci` reports

> npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.

`npm list querystring` shows

```
`-- @node-wot/browser-bundle@0.8.0 -> .\packages\browser-bundle
  `-- browserify@16.5.0
    `-- url@0.11.0
      `-- querystring@0.2.0
```

There is a new versions (16.5.2 and 17.0.0) for browserify.
I will try to have a look!

**Comment by relu91** [comment]
since #690 I was thinking of moving to webpack... idk which are the implications but, I'm feeling that `browserify` is becoming less popular/mantained. On the other hand, webpack is pretty powerful for what we actually need, it might be an overkill

**Comment by JKRhb** [comment]
At the moment, I do not see this warning when running `npm ci` – do you think the issue should be kept open or could it be closed in the meantime?

**Comment by danielpeintner** [comment]
Yes, let's close the issue. I don't get the warning either anymore.

Moreover

`npm list querystring`

reports 

`-- (empty)`

which indicates that the implicit dependency has been removed.
